### DISCUSSION TITLE
翻译不出，一直转圈

### DISCUSSION THREAD

**Discussion by tml633** [immersive-translate/immersive-translate#2318] [Help]
### 概述 | Summary

你好，想我这个网站里的pdf一直翻译不出，一直转圈，是为什么啊
但是把这个文档下载出来，用PDF文档翻译就可以
是我这太多页，是一翻译不了吗
但不是有部分翻译吗，就不是一下子全部
![IMG_3277](https://github.com/user-attachments/assets/11c741b9-967d-4b58-b7ff-9c2696cf1bd0)

**Comment by keyboard3** [comment]
你这个pdf不适合在线网页翻译，你发现你截图中的圈圈了吗？ 变成了一行一个了，如果你离线放到 https://app.immersivetranslate.com/ ，就可以通过算法自动将这些行识别成一个独立段落翻译

**Reply by tml633** [reply]
请问是说这个样子吗
![IMG_3278](https://github.com/user-attachments/assets/070d86f5-2054-4bc6-bdb9-bfb07648a740)

**Reply by keyboard3** [reply]
是的
